src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -23,6 +23,10 @@
     /** The directory where JSON files used to create data bundles are stored. */
     public static final String TEST_DATA_FOLDER = "src/test/resources/data";
 
+    /** The directory where credentials used in Gmail API are stored. */
+    @SuppressWarnings("WeakerAccess")
```

Comment:
Well, I don't think such warnings are invalid ones, and if they are irrelevant they should be suppressed. I don't think adding a new warning helps just because there are already several in the project.

There is no point to having warnings if the project ignores them completely, the project should in fact slowly get rid of them, whether it is through valid suppressions or changing the code itself.